theano/tensor/tests/test_subtensor.py

```diff
@@ -97,18 +98,17 @@ def setUp(self):
         Subtensor.debug = False
         utt.seed_rng()
 
-    def eval_output_and_check(self, t, list=False, mode=None):
+    def eval_output_and_check(self, t, op_type=None, mode=None, length=1):
```

Comment:
Here op_type should be self.sub by default, not None. This probably make many tests in travis fail.